nf_core/__main__.py

```diff
@@ -29,6 +29,31 @@
 # Submodules should all traverse back to this
 log = logging.getLogger()
 
+# Set up nicer formatting of click cli help messages
+rich_click.core.MAX_WIDTH = 100
+rich_click.core.USE_RICH_MARKUP = True
+rich_click.core.COMMAND_GROUPS = {
+    "nf-core": [
+        {
+            "name": "Commands for users",
+            "commands": ["list", "launch", "download", "licences"],
+        },
+        {
+            "name": "Commands for developers",
+            "commands": ["create", "lint", "modules", "schema", "bump-version", "sync"],
+        },
+    ],
+    "nf-core modules": [
+        {
+            "name": "For pipelines",
+            "commands": ["list", "install", "update", "remove"],
+        },
+        {
+            "name": "Developing new modules",
+            "commands": ["create", "create-test-yml", "lint", "bump-versions"],
```

Comment:
Should we mention that one has to be called inside a pipeline repo and one inside the modules repo or will that be too much text?

Reply:
No such thing as too much text now! Instead of being truncated the plug-in will mean that the text just wraps onto a second line if it's long.

We might want to keep it shortish anyway though, still looks better if they're on one line I think.